src/routes.js

```diff
@@ -0,0 +1,2 @@
+export const toMovieListPage = () => "/movielistpage";
```

Comment:
Would it not look better if camelCase was used : ...movieListPage !? just a thought